Automattic/jetpack
# Plugin assets: add spanish version of our banners

## Proposed changes:

WordPress.org allows displaying localized versions of plugin banners, by adding the language code to the existing banner asset URLs. Let's get started with Spanish.

### Other information:


## Jetpack product discussion

Internal reference: p1696242919190359-slack-C01264051NE

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Check that the file names are correct.
* Check for typos in the images.
